apps/system/build/build.js

```diff
@@ -146,6 +146,35 @@ SystemAppBuilder.prototype.enableFirefoxSync = function(options) {
   preprocessor.execute(options, 'FIREFOX_SYNC', fileList);
 };
 
+SystemAppBuilder.prototype.disableBluetoothFeatures = function(options) {
+  var fileList = {
+    process: [
+      ['index.html'],
+      ['js', 'quick_settings.js']
+    ],
+    remove: {
+      'ifdef': [
+        ['js', 'bluetooth.js'],
+        ['js', 'bluetooth_core.js'],
+        ['js', 'bluetooth_headphone.js'],
+        ['js', 'bluetooth_icon.js'],
+        ['js', 'bluetooth_transfer.js'],
+        ['js', 'bluetooth_transfer_icon.js'],
+        ['js', 'bluetooth_transfer_v2.js'],
+        ['test', 'unit', 'bluetooth_core_test.js'],
```

Comment:
test code will not be included in device build, so we don't need to preprocess them